frontend/src/utils/constants.tsx

```diff
@@ -207,7 +213,12 @@ export const polls: Array<SelectablePoll> = [
     withSearchBar: true,
     topBarBackground: null,
     comparisonsCanBePublic: true,
-    extraMetadataOrderBy: ['duration', 'publication_date'],
+    extraMetadataOrderBy: [
+      'duration',
+      'publication_date',
+      'collective_score',
+      'individual_score',
```

Comment:
I'm currently reworking this part.

It works perfectly as is, but we can make those two metadata available for all polls.

The `extraMetadataOrderBy` key is expected to contain only non-generic metadata, i.e. metadata specific to an entity type. For instance the metadata `duration` is specific to the `video` entity type, and may be irrelevant for other types, like the candidates. 

`collective_score` and `individual_score` are generic, it means we use them directly in the file `RatingOrderByInput.tsx`.